R/browse_surveys.R

```diff
@@ -39,15 +39,10 @@ browse_surveys <- function(per_page = 100,
                            include = NULL,
                            folder_id = NULL,
                            oauth_token = get_token()) {
-  if (!is.null(oauth_token)) {
-    u <- "https://api.surveymonkey.com/v3/surveys?"
-    token <- paste("bearer", oauth_token)
-  } else {
-    stop(
-      "Must specify 'oauth_token'.
-      See https://github.com/tntp/surveymonkey#authentication for more info."
-    )
-  }
+
+  u <- "https://api.surveymonkey.com/v3/surveys?"
+  token = get_bearer_token(oauth_token)
+
   if (inherits(start_modified_at, "POSIXct") | inherits(start_modified_at, "Date")) {
```

Comment:
Is it possible to implement the new `format_date()` function here?

Reply:
are you reviewing the latest commit? the branch at most recent commit has 

```
 u <- "https://api.surveymonkey.com/v3/surveys?"
  h <- standard_request_header(oauth_token)

  start_modified_at <- format_date(start_modified_at)
  end_modified_at <- format_date(end_modified_at)
```

at this location, so already done I think